src/Microsoft.ML.Transforms/MissingValueHandlingTransformer.cs

```diff
@@ -15,15 +15,15 @@
 using Microsoft.ML.Transforms.Conversions;
 
 [assembly: LoadableClass(MissingValueHandlingTransformer.Summary, typeof(IDataTransform), typeof(MissingValueHandlingTransformer),
-    typeof(MissingValueHandlingTransformer.Arguments), typeof(SignatureDataTransform),
+    typeof(MissingValueHandlingTransformer.Options), typeof(SignatureDataTransform),
     MissingValueHandlingTransformer.FriendlyName, "NAHandleTransform", MissingValueHandlingTransformer.ShortName, "NA", DocName = "transform/NAHandle.md")]
 
 namespace Microsoft.ML.Transforms
 {
     /// <include file='doc.xml' path='doc/members/member[@name="NAHandle"]'/>
     public static class MissingValueHandlingTransformer
     {
-        public enum ReplacementKind : byte
+        internal enum ReplacementKind : byte
```

Comment:
>internal [](start = 8, length = 8)

the users set an argument of this type in the xtension, think we should leave it public so they can access it.

Reply:
I realized that the entire class can be made internal, this is not an `ITransformer`. I am going to publish another iteration where I revert my changes to this file, and make the class internal.

---
In reply to: [253124498](https://github.com/dotnet/machinelearning/pull/2363#discussion_r253124498) [](ancestors = 253124498)